apps/ledger-live-desktop/src/renderer/components/ProviderIcon/index.tsx

```diff
@@ -13,7 +13,7 @@ export type Props = {
 const ProviderIcon = ({
   name,
   size = "S",
-  boxed = false,
+  boxed = true,
```

Comment:
Just to double check, is this safe? Are there any other teams/parts of the app that are using this component? 😄

Reply:
I will check with the `#live-engineering` slack channel. Thought this could be okay because it enforces all Provider icon to be supported by light and dark mode by default.

Reply:
I think the only place I noticed in LLD is [this](https://github.com/LedgerHQ/ledger-live/blob/9070611ad3500d8879117ab4dd95cbf529ef8cb5/apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx#L902) usage of it.